base/sort.jl

```diff
@@ -407,6 +408,48 @@ julia> searchsortedlast([1, 2, 4, 5, 5, 7], 0) # no match, insert at start
 ```
 """ searchsortedlast
 
+function insorted(x, coll::AbstractVector, lo::T, hi::T, o::Ordering)::Bool where T<:Integer
+    !isempty(searchsorted(coll, x, lo, hi, o))
+end
+
+insorted(x, r::AbstractRange) = in(x, r)
+
+for s in [:insorted]
+    @eval begin
+        $s(x, v::AbstractVector, o::Ordering) = (inds = axes(v, 1); $s(x,v,first(inds),last(inds),o))
+        $s(x, v::AbstractVector;
+           lt=isless, by=identity, rev::Union{Bool,Nothing}=nothing, order::Ordering=Forward) =
+            $s(x,v,ord(lt,by,rev,order))
+    end
+end
+
+
+"""
+    insorted(a, x; by=<transform>, lt=<comparison>, rev=false)
+
+Determine whether an item is in the given sorted collection, in the sense that
+it is [`==`](@ref) to one of the values of the collection. Returns a `Bool`
+value.
+
+# Examples
+```jldoctest
+julia> insorted(4, [1, 2, 4, 5, 5, 7]) # single match
+true
+
+julia> insorted(5, [1, 2, 4, 5, 5, 7]) # multiple matches
+true
+
+julia> insorted(3, [1, 2, 4, 5, 5, 7]) # no match
+false
+
+julia> insorted(9, [1, 2, 4, 5, 5, 7]) # no match
+false
+
+julia> insorted(0, [1, 2, 4, 5, 5, 7]) # no match
+false
+```
```

Comment:
I like the docstring. Needs a compat annotation, like this:
https://github.com/JuliaLang/julia/blob/f13e11c130a59a5d69d17d1e205b13c0564d89d7/base/strings/unicode.jl#L47-L54
If you like, you can also add  a line like ```See also [`searchsorted`](@ref), [`in`](@ref)```. And also links back from those docstrings.